code/obj/table.dm

```diff
@@ -873,7 +866,7 @@
 
 	onUpdate()
 		..()
-		if (the_table == null || the_tool == null || owner == null || get_dist(owner, the_table) > 1)
+		if ((the_table == null || the_tool == null || owner == null || get_dist(owner, the_table) > 1) && !istype(owner, /mob/living/critter/flock/drone)) //allows flock drones to decon flock tables
```

Comment:
Can you explain this a bit more? Comment's not the most clear.

Reply:
huh. cant exactly remember _why_ i did it. i think that has something to do with the action starting but ending onUpdate. ill check if this causes flockdrones to be able to walk away from table disassembly. if it doesnt ill update the comment to be more clear. if it does ill have to work out the problem

Reply:
yeah moving away causes the action to stop

Reply:
Changed out the code to use a custom central flock_decon action instead of trying to reuse the tool interact action